opp_msgs_serialization/include/opp_msgs_serialization/binary_serialization.h

```diff
@@ -0,0 +1,56 @@
+#ifndef MESSAGE_SERIALIZATION_BINARY_SERIALIZATION_H
```

Comment:
Most of these header guards don't follow naming convention, please see [here](https://google.github.io/styleguide/cppguide.html#The__define_Guard) for info on header guard naming practices

Reply:
Updated.